evalml/tests/data_checks_tests/test_highly_null_data_check.py

```diff
@@ -155,8 +177,84 @@ def test_highly_null_data_check_warnings():
     }
 
 
+def test_highly_null_data_check_separate_rows_cols():
+    data = pd.DataFrame(
+        {
+            "lots_of_null": [None, None, None, None, 5],
+            "all_null": [None, None, None, None, None],
+            "no_null": [1, 2, 3, 4, 5],
+        }
+    )
+    row_null_check = HighlyNullDataCheck(
+        pct_null_col_threshold=0.9, pct_null_row_threshold=0.0
+    )
+    highly_null_rows = SeriesWrap(pd.Series([2 / 3, 2 / 3, 2 / 3, 2 / 3, 1 / 3]))
+    validate_results = row_null_check.validate(data)
+    validate_results["warnings"][0]["details"]["pct_null_cols"] = SeriesWrap(
+        validate_results["warnings"][0]["details"]["pct_null_cols"]
+    )
+    assert validate_results == {
+        "warnings": [
+            DataCheckWarning(
+                message="5 out of 5 rows are more than 0.0% null",
+                data_check_name=highly_null_data_check_name,
+                message_code=DataCheckMessageCode.HIGHLY_NULL_ROWS,
+                details={"pct_null_cols": highly_null_rows},
+            ).to_dict(),
+            DataCheckWarning(
+                message="Column 'all_null' is 90.0% or more null",
+                data_check_name=highly_null_data_check_name,
+                message_code=DataCheckMessageCode.HIGHLY_NULL_COLS,
+                details={"column": "all_null", "pct_null_rows": 1.0},
+            ).to_dict(),
+        ],
+        "errors": [],
+        "actions": [
+            DataCheckAction(
+                DataCheckActionCode.DROP_ROWS, metadata={"rows": [0, 1, 2, 3, 4]}
+            ).to_dict(),
+            DataCheckAction(
+                DataCheckActionCode.DROP_COL, metadata={"column": "all_null"}
+            ).to_dict(),
+        ],
+    }
+
+    col_null_check = HighlyNullDataCheck(
+        pct_null_col_threshold=0.0, pct_null_row_threshold=0.9
+    )
+    highly_null_rows = SeriesWrap(pd.Series([2 / 3, 2 / 3, 2 / 3, 2 / 3, 1 / 3]))
```

Comment:
I feel like I'm missing something, where is `highly_null_rows` used?

Reply:
yea i think this is pasta

Reply:
Great catch, thank you 😅